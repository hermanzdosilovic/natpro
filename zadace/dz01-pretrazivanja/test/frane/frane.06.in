1000
1138 4024 2411 2081 210 3178 4221 3754 4715 688 4697 1404 433 1070 230 4472 4747 4781 3083 4658 1047 669 4244 204 339 1925 3258 1433 478 4881 2326 1018 1027 580 30 278 1908 3283 1378 1319 4646 3324 418 4252 3290 3061 180 3699 770 1976 4523 3105 3144 3152 1435 2453 3109 310 260 4031 4258 3390 293 1101 2489 1355 2400 4427 2507 3706 2759 4865 3125 3853 4531 735 3327 4139 3117 4816 3100 2016 2123 4942 4543 2970 1667 1908 1799 1650 1133 2362 3629 866 4796 2684 246 4220 4324 2433 4958 3835 2832 3382 1142 877 2418 2733 2960 4627 2900 4674 4259 154 2814 1748 1011 3399 2240 1926 572 1293 4234 2520 2211 3064 2721 4074 4716 4946 496 1927 3411 4713 2294 4549 1609 1404 4592 172 2402 4928 4677 3555 833 3073 4647 2763 3538 3090 2538 2865 2110 3777 3140 2000 1644 2883 190 2562 2192 3788 2173 4480 108 4603 3589 1556 1106 231 510 1347 1325 76 2859 4137 31 317 3036 2561 2286 3015 4993 956 4813 4434 3366 1340 3327 3885 898 4965 1414 2859 3522 2578 1256 769 2320 392 1429 2299 3457 75 1919 1487 3929 2904 4748 1316 2198 3305 959 1745 4039 1613 1468 2408 3014 3981 1825 1618 1062 4596 464 628 2686 2166 3312 1284 2175 1039 979 767 4242 4687 727 3485 2532 3671 4607 3735 3373 2507 1096 2256 2983 3162 1584 3608 4166 4739 1360 4320 2218 195 361 3254 879 3985 4963 3946 1489 1101 1574 3347 1807 4499 1964 2349 4486 3584 2104 2168 220 3767 1822 3075 398 4230 4381 1996 878 4230 972 3383 602 2195 4360 1495 1204 855 3547 3493 612 3207 4236 4609 1851 4897 10 828 3768 3759 3187 591 2579 3102 4632 623 3841 165 3037 602 844 3541 3838 3846 4397 1316 1696 530 1628 2439 233 4158 4302 1292 2085 2869 515 73 2551 1808 4171 1159 1287 4120 927 3093 3342 4931 3167 3185 2799 3374 643 2341 2207 1922 706 2207 2722 162 815 1186 531 4806 632 3660 439 2046 3265 565 3366 3705 1464 1101 1487 700 3544 3963 3594 1233 3991 1781 2871 2738 2664 1203 290 4761 1931 2105 1945 1562 1690 3485 3626 1025 3213 4772 3316 3006 4615 2117 3448 3324 1431 953 2214 2285 4557 335 137 4301 2553 783 3199 667 1462 1339 3157 3135 2220 1673 1349 8 2623 250 1455 1923 3500 2673 4281 191 391 4504 718 134 3994 4229 4687 4792 807 1197 3987 2300 2053 223 1455 1844 3163 3803 2373 24 181 3467 4313 773 4794 4208 3719 890 3499 553 485 839 3411 1783 853 816 4009 1362 4903 4740 4843 3396 2085 869 1766 728 4324 3887 1949 772 2155 1997 1419 1871 2497 527 3516 3612 2240 2259 3203 1293 2899 1746 2652 775 873 3355 428 388 2042 4340 2815 1577 4052 3281 916 731 731 464 3876 862 291 4564 4169 792 1520 2822 4188 4937 470 2116 3777 1872 173 2647 525 3087 1825 1295 841 1216 373 3112 2498 1505 3413 2794 3348 4683 1803 3404 2324 3267 2567 3048 4105 3366 1584 1748 2890 2669 1748 2781 4502 60 454 1444 4071 1205 2210 3522 681 4484 2436 4847 844 536 3846 158 3238 4679 4766 2249 3413 3538 3931 3984 3961 1512 2631 2436 1513 2393 212 940 2023 1788 3666 4371 3130 4605 1924 4298 4640 3045 4843 932 2002 2480 3897 1972 3814 4014 1207 687 4585 3873 2420 4382 4244 335 1423 297 4062 2271 2256 4714 3598 4354 770 1507 4926 4491 2645 215 3103 1926 4971 597 1037 1232 2817 4585 883 2409 1359 2897 1946 1818 313 1923 3144 1968 3960 640 827 2324 1773 4441 2145 3959 118 1929 1279 4072 2033 722 4137 2450 3050 2354 686 3547 2639 1959 4740 944 1732 3160 2882 4442 1687 3418 3964 1782 4 4983 115 1772 2872 3211 2811 1521 1275 3715 3197 3374 4376 985 1006 3588 1586 4389 4525 4949 1111 552 1505 3186 2598 4117 1572 4494 1247 2955 3053 3500 3255 3356 1647 2148 2642 600 2727 118 2260 3666 3647 2344 1955 329 2332 1512 3465 4202 630 325 4512 810 2577 679 336 339 4624 960 4184 1065 1596 642 4764 3756 3935 3421 3228 669 1813 550 3968 463 3398 4833 4731 3902 3695 3413 3621 324 4963 500 2584 510 3234 4938 100 3558 2507 375 4747 1709 537 2461 40 1231 3334 4583 4124 124 4720 2436 199 4267 2423 2323 4990 1329 266 2158 1357 880 3511 3791 4673 1860 2946 1143 3627 254 2568 3391 997 3081 4928 1650 116 1896 69 150 1280 3600 4053 456 4870 2897 1201 3165 2769 1824 1975 2048 2592 3712 2521 4079 909 431 1934 4467 4748 3501 2602 3111 875 4950 4530 1304 4110 4988 2930 40 967 2095 1001 4933 4119 653 1195 4693 1780 2820 4231 2214 3788 1380 784 4731 4525 126 3440 20 3201 4637 4259 1339 3449 1834 172 114 2324 3088 2303 1684 3684 2098 201 822 1343 3263 3285 1671 1010 2188 3887 3788 2001 3896 723 652 522 3246 2786 1895 3116 1313 4535 3472 3958 3001 4069 1500 1774 1665 567 3854 4036 3031 870 4972 3850 1366 2505 3981 1446 3752 1885 3376 2503 4929 2160 3137 4147 3413 2051 2996 1471 4133 4659 1613 1702 2614 20 3389 2976 802 1565 4751 772 422 657 4810 3510 4029 3968 309 1381 1996 3332 3335 2139 3955 2528 3198 3620 480 184 3530 3999 1453 3935 1797 2053 1764 4757 2508 1520 1395 2794 2641 179 3493 4850 3525 1756 547 4969 4457 2135 4974 2336 908 2299 1819 2724 3872 20 1941 2065 389 3756 3902 191 3762 2023 4673 3124 3883 873 3355 1472 4528 3272 4284 4299 1304
