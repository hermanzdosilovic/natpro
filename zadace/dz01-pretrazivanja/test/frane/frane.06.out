2085980943.100
